src/pages/TrueSightV2/components/TokenOverview.tsx

```diff
@@ -333,6 +337,7 @@ export const TokenOverview = ({ data, isLoading }: { data?: ITokenOverview; isLo
                       </>
                     )
                   }
+                  disableTooltip={data?.symbol === 'KNC'}
```

Comment:
only knc ?

Reply:
KNC should not show kyberscore like stablecoins, but the tooltip text is only for stablecoins, so only hide tooltip for KNC